layouts/shortcodes/card.html

```diff
@@ -13,30 +13,18 @@
 {{/* Image processing options */}}
 {{- $method := .Get "method" | default "Resize" | humanize -}}
 {{- $options := .Get "options" | default "800x webp q80" -}}
+{{- $process := .Get "process" | default (delimit (slice $method $options) " " " ") -}}
```

Comment:
```suggestion
{{- $process := .Get "process" | default (printf "%s %s" $method $options) -}}
```

Guess we can just use [printf](https://gohugo.io/functions/fmt/printf/) here?